pkg/util/util.go

```diff
@@ -43,3 +49,16 @@ func Template(tmpl string, f template.FuncMap, cs *acsapi.ContainerService, c *c
 
 	return b.Bytes(), nil
 }
+
+// GetKubeconfigFromV1Config takes a v1 config and returns a kubeconfig
+func GetKubeconfigFromV1Config(kc *v1.Config) (clientcmd.ClientConfig, error) {
```

Comment:
As per the comment above regarding the anti-pattern, should this function move into the `healthcheck.go`?  Seems like a useful function when you don't have a kubeconfig available.